docs/source/getting-started.rst

```diff
@@ -231,8 +231,8 @@ Item validation
 
 Item validators allows you to match your returned items with predetermined structure
 ensuring that all fields contains data in the expected format. Spidermon allows
-you to choose between schematics_ or `JSON Schema`_ to define the structure
-of your item.
+you to choose from schematics_,`JSON Schema`_ or cerberus_  as the validation tool to
```

Comment:
I always keep the sentence structure first, that's all. At most places, Cerberus is kept capitalized just because it's actually a name. I am sure we all know the origin. Cerberus in capitals coming right up.